draft-js-focus-plugin/src/index.js

```diff
@@ -33,6 +33,35 @@ export default (config = {}) => {
       }
       return 'not-handled';
     },
+    handleKeyCommand: (command, editorState, { setEditorState }) => {
+      if (command === 'backspace' && focusableBlockIsSelected(editorState, blockKeyStore)) {
+        const selection = editorState.getSelection();
+        const content = editorState.getCurrentContent();
+        const startKey = selection.getStartKey();
+        const blockAfter = content.getBlockAfter(startKey);
+        const blockMap = content.getBlockMap().delete(startKey);
+        const withoutAtomicBlock = content.merge({
```

Comment:
hm - I'm wondering whether this should sit inside the focus plugin - or rather inside the plugins editor as a patch over draft-js